missioncontrol/v0/accesses.py

```diff
@@ -154,6 +154,8 @@ def access_id(self):
         )
 
     def iter_track(self, step=5):
+        if step is None:
```

Comment:
This shouldn't be needed. You can specify the default in the schema.

Reply:
Bah this snuck in. But good to know!

Reply:
On second thought, I'd prefer to leave this here. If default is somehow left off higher levels you might end up with a `None` kwarg (as we don't want to specify the default on every level of this function tree)

Reply:
And 3rd thoughts, made it a constant.